test/extension.test.ts

```diff
@@ -184,6 +184,17 @@ suite("json2ts Tests", () => {
         assert.equal(result, ts);
     });
 
+    test("Convert extensive JSON-Value to TypeScript Interfaces while keeping upper and lower cases of first character", () => {
```

Comment:
Shouldn't this just be `while keeping upper cases of first character`? We're not testing for lower case in this test right?